src/pages/patientView/pathwayMapper/PatientViewPathwayMapper.tsx

```diff
@@ -170,6 +170,7 @@ export default class PatientViewPathwayMapper extends React.Component<
                         tooltip: <span>Number of Genes Matched</span>,
                     },
```

Comment:
Let's just define a custom render function here for the score column. This way, you wouldn't need to modify the default render function inside `PathwayMapperTable`, and you wouldn't need to define another prop like `convertScoreRepresentation`.

```suggestion
                        render: (d: IPathwayMapperTable) => (
                            <span>
                                <b>{d.score.toFixed(0)}</b>
                           </span>
                       ),
                    },
```

Reply:
I didn't know I could do that :)

Reply:
no worries :)